ui/src/react-admin/modules/user/helpers/generate-filter-where-object-users.ts

```diff
@@ -174,6 +174,16 @@ export const generateWhereObjectArchief = (
 			});
 		}
 
+		if (!isNil(filters.isKeyUser)) {
+			andFilters.push(
+				...getBooleanFilters<Partial<UserTableState>>(
+					filters,
+					['isKeyUser', 'isException'],
+					['is_key_user', 'is_exception']
+				)
+			);
+		}
```

Comment:
Is exception is only for avo
![image](https://user-images.githubusercontent.com/1710840/229153015-5ca74223-ffc0-4aee-a815-6a9469679abe.png)

```suggestion
		if (!isNil(filters.isKeyUser)) {
			andFilters.push(
				...getBooleanFilters<Partial<UserTableState>>(
					filters,
					['isKeyUser'],
					['is_key_user']
				)
			);
		}
```